### PR TITLE
Porting to Intel

### PR DESCRIPTION
**Pull request by tesonep**

### CODE REVIEW

File: smalltalksrc/VMMakerTests/UnicornX64Simulator.class.st [hunk 1]
```diff
@@ -488,8 +488,32 @@ UnicornX64Simulator >> temporaryRegister [
 	^ UcX86Registers rax
 ]
 
+{ #category : #accessing }
+UnicornX64Simulator >> vectorRegister0Value [
```
Comment (nrainhart) [inline smalltalksrc/VMMakerTests/UnicornX64Simulator.class.st:492]: This is the same as `xmm0` right? Maybe we can keep only one? Or implement one using the other

File: smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st [hunk 2]
```diff
@@ -1398,16 +1400,19 @@ StackToRegisterMappingCogit >> genAddFloat32Vector [
...
-	array2Reg := self allocateVectorRegForStackEntryAt: 1 notConflictingWith: (self registerMaskFor: array1Reg).
-	sumReg := self allocateVectorRegNotConflictingWith: 0.
+	objectMemory wordSize = 4 ifTrue: [ ^ EncounteredUnknownBytecode ].	
 	
-	self ssTop moveToVectorReg: array1Reg.
-	self ssPop: 1.
+	array2Reg := self allocateVectorRegForStackEntryAt: 0 notConflictingWith: 0.
+	array1Reg := self allocateVectorRegForStackEntryAt: 1 notConflictingWith: (self registerMaskFor: array2Reg).
+	"The result register is the same that the array1Reg"
+	sumReg := array1Reg.
```
Comment (nrainhart) [inline smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st:1409]: Should `genAddFloat32Vector` change as well?

**Review by nrainhart — APPROVED** [review]
Goes out with 🍟 !

File: smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st (hunk 2 above)
Comment (tesonep) [reply smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st:1409]: I have changed... I should push the changes maybe.

File: smalltalksrc/VMMakerTests/UnicornX64Simulator.class.st (hunk 1 above)
Comment (tesonep) [reply smalltalksrc/VMMakerTests/UnicornX64Simulator.class.st:492]: Yes and no, the vectorRegister0Value is polymorphic with other machine simulators

File: smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st (hunk 2 above)
Comment (tesonep) [reply smalltalksrc/VMMaker/StackToRegisterMappingCogit.class.st:1409]: I was pushing to other repo.... :P

**Pull request merged**